src/multiqc/config.vsh.yaml

```
@@ -54,25 +54,21 @@ argument_groups:
       - name: "--include_modules"
         type: string
         multiple: true
-        multiple_sep: ","
-        example: fastqc,cutadapt
+        example: fastqc;cutadapt
```

I overlooked it last time; I suggest using arrays:

```suggestion
        example: [fastqc, cutadapt]